dev-docs/windows-dev-setup.md

```diff
@@ -0,0 +1,462 @@
+
+The previous tests have mostly to make sure that the linters are all installed, before running them as intended during dev.
+
+To do this the venv path must be modified to include the root of the checked out source.
+In my case
+
+```shell
+C:\mewbot_dev\mewbot
+```
+
+There are a number of ways to do this.
+
+#### Use a .pth files
+
+Probably the best way is to use a `.pth` file - these provide instructions to the python interpreter as to additional folders to include in its path.
+
+A path file is just a text file with extension `.pth` which include folders you want added to the python path - one per line.
+
+It must be placed on the path of the python interpreter your currently using. In this case, assuming you have the same paths as me, probably the best place to put it would be
+
+```shell
+C:\mewbot_dev\mewbot_venv\Lib\site-packages
+```
+
+Simple add a file named `mewbot.pth` here with the single line `C:\mewbot_dev\mewbot`. It's probably best to add it as a new file, instead of modifying one of the existing ones - as these are under system control and may be overwritten without warning.
+
+#### Modify the venv activation script
+
+This is probably the harder option, but is included for completeness.
+
+In my case, the script could be found at
+
+```shell
+C:\mewbot_dev\mewbot_venv\Scripts\activate.bat
+```
+
+and looked like this
+
+```shell
+@echo off
+
+rem This file is UTF-8 encoded, so we need to update the current code page while executing it
+for /f "tokens=2 delims=:." %%a in ('"%SystemRoot%\System32\chcp.com"') do (
+    set _OLD_CODEPAGE=%%a
+)
+if defined _OLD_CODEPAGE (
+    "%SystemRoot%\System32\chcp.com" 65001 > nul
+)
+
+set VIRTUAL_ENV=C:\mewbot_dev\mewbot_venv
+
+if not defined PROMPT set PROMPT=$P$G
+
+if defined _OLD_VIRTUAL_PROMPT set PROMPT=%_OLD_VIRTUAL_PROMPT%
+if defined _OLD_VIRTUAL_PYTHONHOME set PYTHONHOME=%_OLD_VIRTUAL_PYTHONHOME%
+
+set _OLD_VIRTUAL_PROMPT=%PROMPT%
+set PROMPT=(mewbot_venv) %PROMPT%
+
+if defined PYTHONHOME set _OLD_VIRTUAL_PYTHONHOME=%PYTHONHOME%
+set PYTHONHOME=
+
+if defined _OLD_VIRTUAL_PATH set PATH=%_OLD_VIRTUAL_PATH%
+if not defined _OLD_VIRTUAL_PATH set _OLD_VIRTUAL_PATH=%PATH%
+
+set PATH=%VIRTUAL_ENV%\Scripts;%PATH%
+set VIRTUAL_ENV_PROMPT=(mewbot_venv) 
+
+:END
+if defined _OLD_CODEPAGE (
+    "%SystemRoot%\System32\chcp.com" %_OLD_CODEPAGE% > nul
+    set _OLD_CODEPAGE=
+)
+
+```
+
+We need to modify the `PYTHONPATH` variables so statements such as
+```python
+from tools.common import Annotation, ToolChain
+```
+
+in `lint.py` will function.
+
+Add a command such as
+
+```shell
+set PYTHONPATH=C:\mewbot_dev\mewbot;%PYTHONPATH%
+```
+
+to your `activate.bat` file - ideally somewhere between
+
+```shell
+@echo off
+...
+:END
+```
+
+so it won't be printed every time you activate the venv.
+Save and close all open terminals.
+Then open a new terminal, activate the venv, and check that `tools` can now be found by doing
+
+```shell
+(mewbot_venv) C:\Users\Adriann Ves Bloche>python
```

Comment:
check; accidental name inclusion?

Reply:
Should be connected - thanks for the catch